docs/reference/commandline/build.md

```diff
@@ -251,3 +252,22 @@ flag](/reference/run/#specifying-custom-cgroups).
 Using the `--ulimit` option with `docker build` will cause each build step's 
 container to be started using those [`--ulimit`
 flag values](/reference/run/#setting-ulimits-in-a-container).
+
```

Comment:
I think a heading here would make it stand out better from the examples, `## Build-time variables`, or (if we see this as part of the examples) a heading-3; `### Build-time variables`